tests/e2e/test_rosacli_cluster.go

```diff
@@ -21,6 +21,13 @@ import (
 	"github.com/openshift/rosa/tests/utils/profilehandler"
 )
 
+const (
+	StageURL      = "https://console.dev.redhat.com/openshift/details/s/"
+	ProductionURL = "https://console.redhat.com/openshift/details/s/"
+	StageEnv      = "https://api.stage.openshift.com"
+	ProductionEnv = "https://api.openshift.com"
+)
```

Comment:
This should also be put in common function.
I am also surprised that this does not exist yet for the API URLs